### PR TITLE
fix: removed unused sources and database backends

### PR DESCRIPTION
**Pull request by evadnoob**
we're not going to maintain these various sources and backends on our
fork, time to diverge

### CODE REVIEW

File: database/cockroachdb/README.md [hunk 1]
```diff
@@ -1,19 +0,0 @@
-# cockroachdb
-
-`cockroachdb://user:password@host:port/dbname?query` (`cockroach://`, and `crdb-postgres://` work, too)
-
-| URL Query  | WithInstance Config | Description |
-|------------|---------------------|-------------|
```
Comment (masterkidan) [inline database/cockroachdb/README.md:6]: Can you keep this ?Might be useful as we have an investigation for cockroach planned at some point

File: database/mysql/README.md [hunk 2]
```diff
@@ -1,55 +0,0 @@
-# MySQL
-
-`mysql://user:password@tcp(host:port)/dbname?query`
```
Comment (masterkidan) [inline database/mysql/README.md:3]: Any thoughts we might end up using this for migrations of flagship?

File: database/pgx/README.md [hunk 3]
```diff
@@ -1,39 +0,0 @@
-# pgx
```
Comment (masterkidan) [inline database/pgx/README.md:1]: Why did we remove this? This is just using the pgx driver for postgres migrations... might be useful to keep around.

**Review by samnguyen-outreach — APPROVED** [review]

File: database/spanner/README.md [hunk 4]
```diff
@@ -1,46 +0,0 @@
-# Google Cloud Spanner
```
Comment (masterkidan) [inline database/spanner/README.md:1]: do we need to keep this given our gcp commitments?

File: database/mysql/README.md (hunk 2 above)
Comment (evadnoob) [reply database/mysql/README.md:3]: We would never :-) We have several other well used migration tools

File: database/cockroachdb/README.md (hunk 1 above)
Comment (evadnoob) [reply database/cockroachdb/README.md:6]: We can always refer back to the original source if we need it. I'd prefer to not have to maintain the tests and everything that we don't support. 

I'm just finding a lot problems like poor internal contracts on the Handler type and the testing package failing on different sources and backends, and leaving running docker containers around.

File: database/pgx/README.md (hunk 3 above)
Comment (evadnoob) [reply database/pgx/README.md:1]: Same, still have the original as reference

**Pull request merged**